cfgov/jinja2/v1/events/_macros.html

```diff
@@ -202,12 +202,16 @@ <h2 class="event-venue_heading">
             <tr>
                 <td>
                     {% import 'macros/time.html' as time %}
-                    {{ time.render(
-                        bound.start_time.render(),
-                        {'date':false,'time':true,'timezone':false}
-                    ) }}
-                    &ndash;
-                    {{ time.render(bound.end_time.render(), {'time':true, 'timezone':true}) }}
+                    {% if bound.start_time | string != 'None' %}
```

Comment:
I believe `{% if bound.start_time.value is not none %}` will work okay here, because `start_time` is actually the Wagtail block, not the raw value. I tested this locally and it seems to work for me.